src/databricks/labs/ucx/source_code/python_linter.py

```diff
@@ -270,6 +270,9 @@ def get_constant_path(self) -> str | None:
         return None
 
 
+P = TypeVar("P", bound=Callable)
```

Comment:
convention is to use `T` for the type var

Reply:
We already have T earlier in the filem with a different binding